modules/docs/mdx/changelog.stories.mdx

```diff
@@ -11,5 +10,4 @@ import Markdown from '../../../utils/storybook/Markdown.tsx';
     },
   }}
 />
-
```

Comment:
This empty line needs to be put back, it removes the Changelog from the Storybook navigation. See: https://github.com/Workday/canvas-kit/pull/1962/files#r1115267605